### DISCUSSION TITLE
Как в шаблон печати добавить изображение?

### DISCUSSION THREAD

**Discussion by Ekhorikov2** [totumonline/totum-ru-issues-and-discussions#400] [Как это сделать?] [answered]
Всем привет.
Подскажите, как в шаблон печати (КП, счёта и т.д.) добавить изображение? Например логотип компании в хедер КП? Можно через HTML шаблона, но как-то может можно брать изображение из самого TOTUM?

**Comment by erpic-ru** [comment, marked as answer]
Заливаете через поле файл, смотрите адрес ссылки и ставите ее в код печатной формы:
`<img src="https://url сайта с тотумом/fls/6_h_shapka_pechati.png">`

**Reply by Ekhorikov2** [reply]
В HTML, я правильно понимаю?

**Reply by Ekhorikov2** [reply]
Всё получилось. Спасибо за помощь!
Если кому-то нужно будет, в коде это вот так выглядит:

![image](https://github.com/totumonline/totum-ru-issues-and-discussions/assets/112487577/33b56f91-22df-46b9-bbd9-5855023e19f4)

padding-top - отступ от верхней части документа.
